params/params_testpownet.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2017-2018 The qitmeer developers
```

Comment:
We can remove the params file so that we don't need to bother with the address encoding magics. You have to FIX them if you want the additional params file anyway. The current values are not correct since they are copied&pasted from the test-net params file.